component/helm_values.jsonnet

```
@@ -0,0 +1,22 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local prometheus = import 'lib/prometheus.libsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.json_exporter;
+
+local labels =
+  if std.member(inv.applications, 'prometheus') then
+    prometheus.RegisterNamespace({}).metadata.labels
+;
```

You'll probably want to make sure that this is an empty object and not `null` when component prometheus isn't present:

```suggestion
local labels =
  if std.member(inv.applications, 'prometheus') then
    prometheus.RegisterNamespace({}).metadata.labels
  else {}
;
```